python/ray/autoscaler/ray-schema.json

```diff
@@ -1,281 +1,387 @@
 {
-	"$schema": "http://json-schema.org/draft-07/schema#",
-	"$id": "http://github.com/ray-project/ray/python/ray/autoscaler/ray-schema.json",
-	"title": "Ray AutoScaler",
-	"description": "Ray autoscaler schema",
-	"type": "object",
-	"definitions": {
-        "commands": {
-            "type": "array",
```

Comment:
are there any major changes in this file other than tab/space conversions?

Reply:
[Only meaningful change](https://github.com/ray-project/ray/pull/9515/commits/42800f8c8948cf69ceafa5052459df8df145dc9b)